publishable/config/voyager.php

```diff
@@ -94,21 +95,19 @@
     */
 
     'widgets' => [
-        [
-            'name'  => 'User',
+        'user' => [
             'icon'  => 'voyager-group',
             'model' => TCG\Voyager\Models\User::class,
             'url'   => 'admin/users',
             'image' => '/images/widget-backgrounds/02.png',
         ],
```

Comment:
i thought that the model is the one responsible for declaring the view and what logic gets sent to, so even with the new approach , nothing changed except its now been enforced on the user.

however on a side note, if u absolutely want to go with the new approach, then the user can simply extend the class and make his changes (same with controller, same with models)